src/expandableCalendar/commons.ts

```diff
@@ -8,15 +8,16 @@ export const isIos = Platform.OS === 'ios';
 export const screenWidth = width;
 export const screenHeight = height;
 export const screenAspectRatio = screenWidth < screenHeight ? screenHeight / screenWidth : screenWidth / screenHeight;
+// @ts-expect-error
```

Comment:
I think the proper way to cast it is like that: 
`(Platform as PlatformIOSStatic).isPad`

(import PlatformIOSStatic as well)

Reply:
Thanks!